jenkins/test-image/Dockerfile

```diff
@@ -43,6 +43,13 @@ RUN apt-get -o Acquire::Check-Valid-Until=false update && apt-get install -y \
 	--no-install-recommends \
 	&& rm -rf /var/lib/apt/lists/*
 
+# Download and symlink etcd. We need this for verification and integration tests.
+RUN export ETCD_VERSION=v3.0.4; \
```

Comment:
hack/install-etcd.sh from HEAD is packaged in an image and used in all branch testing?